File: lista1.cpp
#include <iostream>
#include <cstdlib>
#include <algorithm>
#include <ctime>
#include <vector> //para a lista

using namespace std;

#include "biblioteca.h"

int main(){
    vector<int> listaNumeros;
    int quantidadeNumeros;
    int faixaInicial = 10, faixaFinal = 50;
    vector<int> listaResultado;

    cout << "Digite quantos numeros  quer gerar: ";
    cin >> quantidadeNumeros;

    popularlistaAleatoria(listaNumeros, quantidadeNumeros, faixaInicial, faixaFinal);
    exibirLista(listaNumeros);

    copiarlistasemReplicados(listaNumeros, listaResultado);
    exibirLista(listaResultado);

    return 1;
}

File: lista1.py

from biblioteca import popular_lista_aleatoria, exibir_lista, copiar_lista_sem_replicados


#declarando uma lista em python
lista_numeros = []

#recebendo a quantidade de números a serem gerados aleatoriamente
quantidade_numeros = int(input("Digite quantos números gostaria de armazenar: "))

#definindo a favia de intervalo dos números a serem sorteados
faixa_inicial = 10
faixa_final = 50

popular_lista_aleatoria(lista_numeros, quantidade_numeros, faixa_inicial, faixa_final)

exibir_lista(lista_numeros)


lista_resultado = []
copiar_lista_sem_replicados(lista_numeros, lista_resultado)
lista_resultado.sort()
exibir_lista(lista_resultado)